src/matlab2tikz.m

```diff
@@ -1613,18 +1613,15 @@ function assertRegularAxes(handle)
 % not too fancy plots.
     str = '';
 
-    if ~isVisible(h)
-        return
-    end
-
```

Comment:
Generally i am rather fond of exiting early. I am sure, that it has no runtime effect at all, but is there any particular reason you put it in the end?